requirements.txt

```diff
@@ -2,11 +2,9 @@ wheel>=0.29
 pandas>=0.24.2
 numpy>=1.18.4
 dash>=1.16.2
-dash-core-components>=1.12.1
-dash-html-components>=1.1.1
```

Comment:
I am not sure to understand why we are erasing these requirements here?

Reply:
This is quite optional, it's more cosmetics than anything else: they are direct dependencies of  `dash` so installing dash installs these :)

Reply:
Oh ok, I thought I had to install them separately when I first installed dash, nice catch!